internal/planner/mapper/mapper.go

```diff
@@ -365,6 +366,15 @@ func resolveAggregates(
 	def client.CollectionDefinition,
 	store client.Store,
 ) ([]Requestable, error) {
+	var collectionShortID uint32
+	if def.Version.CollectionID != "" {
```

Comment:
question: Could it be `""` and what happens if it is?

Reply:
Yes, this surprised me too - I think it is `""` for commit queries and some other edge cases.

Reply:
The `collectionShortID` block is only hit when this function is called with a collection.  It would panic before the `collectionShortID` fetching when retrieving a field description from a nil/empty def